dietpi/dietpi-software

```diff
@@ -8664,8 +8663,7 @@ _EOF_
 			fi
 
 			# Add occ command shortcut, use 'ncc' as 'occ' is reserved for ownCloud:
-			echo -e '#!/bin/bash\nsudo -u www-data php /var/www/nextcloud/occ $*' > /usr/local/bin/ncc
-			chmod +x /usr/local/bin/ncc
+			sed -i 's/#alias ncc/alias ncc/' /DietPi/dietpi/func/dietpi-globals
```

Comment:
```dietpi-globals``` will be replaced during any ```dietpi-updates```. Any of these changes will be lost.

Only thing I can think of is either:
- Always have the alias in place (as long as the command returns 'not found' or similar, should be fine)
- Create a include file ```/var/lib/dietpi/dietpi-globals/alias_include``` which gets loaded with ```dietpi-globals```

Reply:
@Fourdee 
That's true, didn't think about that. I also saw there are already two static non core alias' inside: https://github.com/Fourdee/DietPi/pull/1377/files#diff-03a4faa1a3aeff0af60938482f5791b3R57

`if [ -f /var/www/owncloud/occ ]; then` could be by the way also used within main dietpi-globals (as well as for all other additional software), then there is one file less to check on script loading ;). If we choose a separate file, then I would use commenting, just because we can there, otherwise use dietpi-globals. One other thing is, that globals would need to be reloaded if the command wants to be used within installation/configuration (like in ownCloud/Nextcloud), on the other hand, just use `alias occ="..."` as command there also works until next globals reload. That way er keep it as installation step and can inform users 😉.

I will make some tests about script loading influence with: `time for i in ``seq 1 100``; do ; done`